src/material/slider/slider-input.ts

```diff
@@ -275,7 +275,12 @@ export class MatSliderThumb implements _MatSliderThumb, OnDestroy, ControlValueA
   /** @docs-private */
   initProps(): void {
     this._updateWidthInactive();
-    this.disabled = this._slider.disabled;
+
+    // If this or the parent slider is disabled, just make everything disabled.
+    if (this.disabled !== this._slider.disabled) {
+      this._slider.disabled = true;
```

Comment:
should this just happen on init? and not when the setter for `disabled` triggers?

Reply:
This behavior is already present in the `disabled` setters. If either slider thumb gets disabled or enabled, the entire slider is updated to that state. On init the handling was incorrect bc it didn't consider the fact that the slider inputs could be set to disabled before this point in time

Reply:
Makes sense, thx for the explanation.

Reply:
No problem. For more context, this specific issue happens when someone initializes a form control w/ disabled true

```
control = new FormControl({value: '0', disabled: true});
```